src/main/java/creativei/entity/InquiryEducation.java

```diff
@@ -27,16 +27,17 @@
     @Column(name="mark_scheme")
     private MarkScheme markScheme;
     private String type;
+    public InquiryEducation(){}
 
     public InquiryEducation(EducationVo educationVo) {
         this.educationQualification=EducationQualification.stringToEnum(educationVo.getEducationQualification());
-        this.instituteName=educationVo.getInstituteName()==null?null:educationVo.getInstituteName();
+        this.instituteName=educationVo.getInstituteName()==null||educationVo.getInstituteName().isEmpty()?null:educationVo.getInstituteName();
         this.stream=Stream.stringToEnum(educationVo.getStream());
-        this.status=educationVo.getStatus()==null?null:Status.stringToEnum(educationVo.getStatus());
```

Comment:
It is already null. No need to use tertiary here. 
Use: 
if(educationVo.getStatus()!=null)  this.status= stringToEnum(educationVo.getStatus());